tools/check-response-time.js

```diff
@@ -0,0 +1,115 @@
+const fetch = require("node-fetch");
+
+const BASEURL = `https://design.payex.com/${process.env.basename}`;
+const STATES = {
+    SUCCESS: "#2da944",
+    WARNING: "#ff9f00",
+    ERROR: "#cd2e00"
+};
+
+const MS_THRESHOLDS = {
+    ERROR: 500,
+    WARNING: 300
+};
+
+const urlsToCheck = [
+    {
+        name: "px-script",
+        path: "/scripts/px-script.js"
+    },
+    {
+        name: "px.css",
+        path: "/styles/px.css"
+    }
+];
+
+const asyncForEach = async (array, callback) => {
+    for (let i = 0; i < array.length; i++) {
+        await callback(array[i], i, array);
+    }
+};
+
+const sendSlackMessage = messageData => {
+    fetch(process.env.slack_designguide_webhook, {
+        method: "POST",
+        body: JSON.stringify(messageData),
+        headers: {
+            "Content-Type": "application/json"
+        }
+    })
+        .then(() => {
+            console.log("Reported response time to slack.");
+        })
+        .catch(console.error);
+};
+
+const getResponseTime = async url => {
+    const startTime = new Date().getTime();
+    const endTime = await fetch(url)
+        .then(() => new Date().getTime())
+        .catch(error => {
+            console.error(`Error fetching ${url}: ${error}`);
+        });
+
+    return endTime - startTime;
+};
+
+const checkResponseTime = async () => {
+    const slackMessageData = {
+        text: `Response times for *${process.env.basename} (<https://github.com/PayEx/design.payex.com/commit/${process.env.GitVersion_ShortSha}|${process.env.GitVersion_ShortSha}>)*:`,
```

Comment:
Not sure if this works, but I'd like the `basename` and commit hash to be formatted with code ticks instead of in bold:

```suggestion
        text: `Response times for \`${process.env.basename} (<https://github.com/PayEx/design.payex.com/commit/${process.env.GitVersion_ShortSha}|${process.env.GitVersion_ShortSha}>)\`:`,
```

Reply:
I think it will, lets try